### ISSUE TITLE
Project

### ISSUE THREAD

**Issue by JihyeHaley** [JihyeHaley/PurpleGirl#5]
<%@ page language="java" contentType="text/html; charset=UTF-8"
    pageEncoding="UTF-8" import = "vo.WeetCategoryVO, java.util.ArrayList"
    import = "vo.WeetCustomerInfoVO, java.util.ArrayList" 
    import = "vo.WeetOrderlistVO, java.util.ArrayList"
    import = "vo.WeetVideoVO, java.util.ArrayList" isELIgnored="true"%>
<!DOCTYPE html>
<html>
<head>
<meta charset="UTF-8">
<link href="/weet/resources/styles/welcomesheet.css" rel="stylesheet">
<link href="/weet/resources/styles/style.css" rel="stylesheet">
<link href="/weet/resources/styles/scroll.css" rel="stylesheet">

<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>

<title>weeT</title>

</head>
<body>
  

  
  
  
<nav class="main-nav transparent stick-fixed">
<div class="container relative clearfix">

<div class="nav-logo-wrap local-scroll">
<a href="#top" class="logo">
<img src="/default/img/logo.png" alt="" />
</a>
</div>
<div class="mobile-nav">
<i class="fa fa-bars"></i>
</div>

<div class="inner-nav desktop-nav">
<ul class="clearlist scroll-nav local-scroll">
	<li class="active"><a href="#home">About Us</a></li>
	<li><a href="#about">Instagram</a></li>
	<li><a href="#channel">Channel</a></li>
</ul>
</div>
</div>
</nav>

    
   
<main>

       

<div id="pbar-wrapper">
  <div id="pbar"></div>
</div>
<div id="page">
  
  <div id="poem"></div>
</div>

</main>
<section>
   <div class="video"><iframe width="440" height="315" src="https://www.youtube.com/embed/6Cp_4UejhrM" style="margin-bottom:50px;"></iframe></div>
   <div class="video"><iframe width="440" height="315" src="https://www.youtube.com/embed/sVW_ueF7-A0" style="margin-bottom:50px;"></iframe></div>
   <div class="video"><iframe width="440" height="315" src="https://www.youtube.com/embed/5gaInOuEp74" style="margin-bottom:50px;"></iframe></div>
</section>
   

<div class="grid-container">
  <div class="individual">
    <img class='grid-item grid-item-1' src='https://images.unsplash.com/photo-1544568100-847a948585b9?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=400&fit=max&ixid=eyJhcHBfaWQiOjE0NTg5fQ' style="width:285px; height:auto;"alt=''>
    <p>"I'm so happy today!"</p>
  </div>
  <div class="individual">
    <img class='grid-item grid-item-2' src='https://images.unsplash.com/photo-1517423440428-a5a00ad493e8?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=400&fit=max&ixid=eyJhcHBfaWQiOjE0NTg5fQ' style="width:285px; height:auto;" alt=''>
    <p>"I see those nugs."</p>
  </div>
  <div class="individual">
    <img class='grid-item grid-item-3' src='https://images.unsplash.com/photo-1510771463146-e89e6e86560e?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=400&fit=max&ixid=eyJhcHBfaWQiOjE0NTg5fQ' style="width:285px; height:auto;" alt=''>
    <p>"I love you so much!"</p>
  </div>
  <div class="individual">
    <img class='grid-item grid-item-4' src='https://images.unsplash.com/photo-1507146426996-ef05306b995a?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=400&fit=max&ixid=eyJhcHBfaWQiOjE0NTg5fQ' style="width:285px; height:auto;" alt=''>
    <p>"I'm the baby of the house!"</p>
  </div>
  <div class="individual">
    <img class='grid-item grid-item-5' src='https://images.unsplash.com/photo-1530281700549-e82e7bf110d6?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=400&fit=max&ixid=eyJhcHBfaWQiOjE0NTg5fQ' style="width:285px; height:auto;" alt=''>
    <p>"Are you gunna throw the ball?"</p>
  </div>
  <div class="individual">
    <img class='grid-item grid-item-6' src='https://images.unsplash.com/photo-1548199973-03cce0bbc87b?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=400&fit=max&ixid=eyJhcHBfaWQiOjE0NTg5fQ' style="width:285px; height:auto;" alt=''>
    <p>"C'mon friend!"</p>
  </div>
  <div class="individual">
    <img class='grid-item grid-item-7' src='https://images.unsplash.com/photo-1552053831-71594a27632d?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=400&fit=max&ixid=eyJhcHBfaWQiOjE0NTg5fQ' style="width:285px; height:auto;" alt=''>
    <p>"A rose for mommy!"</p>
  </div>
  <div class="individual">
    <img class='grid-item grid-item-8' src='https://images.unsplash.com/photo-1518717758536-85ae29035b6d?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=400&fit=max&ixid=eyJhcHBfaWQiOjE0NTg5fQ' style="width:285px; height:auto;" alt=''>
    <p>"You gunna finish that?"</p>
  </div>
  <div class="individual">
    <img class='grid-item grid-item-9' src='https://images.unsplash.com/photo-1535930891776-0c2dfb7fda1a?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=400&fit=max&ixid=eyJhcHBfaWQiOjE0NTg5fQ' style="width:285px; height:auto;" alt=''>
    <p>"We can't afford a cat!"</p>
  </div>
  <div class="individual">
    <img class='grid-item grid-item-10' src='https://images.unsplash.com/photo-1504595403659-9088ce801e29?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=400&fit=max&ixid=eyJhcHBfaWQiOjE0NTg5fQ' style="width:285px; height:auto;" alt=''>
    <p>"Dis my fren!"</p>
  </div>
</div>

<script>

const title = 'wee T';
const poet = 'Oliver Haley - 1991 1997';

const poem = `Wee T is a neew platform WebSite 
helping you to quickly make your own 
travel plan Treasure your best plan!`;

let $poem = document.querySelector('#poem'),
    $chars = null,
    current = 0,
    stack = [],
    delayInterval = 4,
    delayIntervalId = null,
    repeatDelaySec = 7,
    repeatDelayId = null,
    autorepeat = true,
    _timer = null;

function run_countdown(pbarEl, sec, callbackEnd, callbackStart) {
  let reverse_counter = 100,
      pbar = document.querySelector(pbarEl),
      _delay = sec * 10;
  
  if (pbar === null) {
    return;
  }
  
  pbar.style.width = '0%';
  clearInterval(_timer);
  
  if (callbackStart !== undefined && typeof callbackStart === 'function') {
    callbackStart();
  }
  
  _timer = setInterval(function() {
    let progress = (100 - --reverse_counter) + '%';

    //window.requestAnimationFrame(() => pbar.style.width = progress);

    if (reverse_counter <= 0) {
      clearInterval(_timer);
      if (callbackEnd !== undefined && typeof callbackEnd === 'function') {
        callbackEnd();
      }
    }
  }, _delay);
}

function prepareText() {
  let poemChars = poem.split(''),
      charLen = poemChars.length,
      i;
  
  for (i = 0; i < charLen; i++) {
    if (poemChars[i] === '\n') {
      poemChars[i] = `<span class="cr">${poemChars[i]}</span>`;
    } else {
      poemChars[i] = `<span class="char">${poemChars[i]}</span>`;
    }
  }

  $poem.innerHTML = poemChars.join('');
  $chars = $poem.querySelectorAll('.char');
}

function styleColor(disturb) {
    const maxLightness = disturb ? 255 : 50,
          r = Math.round(Math.random() * maxLightness),
          g = Math.round(Math.random() * maxLightness),
          b = Math.round(Math.random() * maxLightness);
    return `color: rgba(${r},${g},${b},${Math.random() * 0.5 + 0.5});`;
}

function styleFontWeight() {
  return 'font-weight:' + (Math.random() * 700 + 300) + ';';
}

function styleFontSize(disturb) {
  let baseSize = disturb ? 4 : 1.8,
      rndSize = disturb ? 5 : 0.6;
  return 'font-size:' + (Math.random() * rndSize + baseSize) + 'rem;';
}

function styleFontVariation() {
  return "font-variation-settings: 'slnt' " + (Math.random() * 15 * -1) + 
      ", 'ital' 0.5, 'CASL' " + Math.round(Math.random()) +
      // "'MONO' 1, " +
      ';';
}

function processStack() {
  if (stack.length) {
    let task = stack.shift();
    window.requestAnimationFrame(() => task.el.style = task.style);
  }
  
  if (stack.length) {
    delayIntervalId = window.setTimeout(processStack, delayInterval);
  } else if (autorepeat) {
    run_countdown('#pbar', repeatDelaySec, styleText);
  }
}

function styleText() {
  $chars.forEach((el, i) => {
    const disturb = Math.random() > 0.98; // 2% extreme values.
    let style = '';
    // style += 'color: #445;'; // Set font color the show the chars.
    style += styleFontWeight();
    style += styleFontSize(disturb);
    style += styleColor(disturb);
    style += styleFontVariation();
    stack.push({el: el, style: style});
  });
  
  processStack();
}

window.addEventListener('click', () => {
  // if (stack.length > 2000) {
  //   stack = [];
  // }
  if (delayIntervalId !== null) {
    window.clearTimeout(delayIntervalId);
    delayIntervalId = null;
  }
  
  if (repeatDelayId !== null) {
    window.clearTimeout(repeatDelayId);
    repeatDelayId = null;
  }
  
  stack = [];
  styleText();
});

function init() {
  // autorepeat = false;
  prepareText();
  styleText();
}

init();

</script>

<div class="container">
<button type="button" class="btn btn-info btn-round" data-toggle="modal" data-target="#loginModal">
    Login
</button>  
<div class="modal fade" id="loginModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
   
  <div class="modal-dialog modal-dialog-centered" role="document" >
    <div class="modal-content" >
      <div class="modal-header border-bottom-0">
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body border-bottom-0">
         <br>
         <div class="form-title text-center">
         <img src="/weet/resources/images/weeT_signup_logo.svg" />    
      </div>
      
        <div class="form-title text-center">
          <h4>Login</h4>
        </div>
        <div class="d-flex flex-column text-center">
          <form>
            <div class="form-group">
              <input type="email" class="form-control" id="email1"placeholder="Your id...">
            </div>
            <div class="form-group">
              <input type="password" class="form-control" id="password1" placeholder="Your password...">
            </div>
            <button type="button" class="btn btn-info btn-block btn-round" onclick="location.href='/weet/signinCheck.do'">Login</button>
            
          </form>
          
          <div class="text-center text-muted delimiter">or use a social network</div>
         <div class="d-flex justify-content-center social-buttons">
            <button type="button" class="btn btn-secondary btn-round" data-toggle="tooltip" data-placement="top" title="Google">
                <img src="/weet/resources/images/btn_google_light_normal_ios.svg">
             </button>
          </div>
        </div>
      </div>
      <div class="modal-footer d-flex justify-content-center ">
        <div class="signup-section text-center">Not a member yet? <a href="#a" class="text-info"> Sign Up</a>.</div>
      </div>
    </div>
      
  </div>
</div>
</div>

</body>

</html>
